### PR TITLE
Kiali Istio dashboards incompatible with thanos-query

### PR DESCRIPTION
**Pull request by aljesusg**
Fix https://github.com/kiali/kiali/issues/4303
Require: https://github.com/kiali/kiali/pull/4386

### CODE REVIEW

File: deploy/kiali/kiali_cr.yaml [hunk 1]
```diff
@@ -623,6 +623,10 @@ spec:
 #      custom_headers: {}
 #      health_check_url: ""
 #      is_core: true
+#      thanos_proxy:
+#        enabled: false
+#        retention_period: "7d"
+#        scrape_interval: "30s"
```
Comment (jmazzitelli) [inline deploy/kiali/kiali_cr.yaml:629]: you should explain these settings above -- around line 607. Just add a comment up there to briefly talk about these so people know what they are for.

**Review by jmazzitelli — CHANGES_REQUESTED** [review]

File: deploy/kiali/kiali_cr.yaml (hunk 1 above)
Comment (aljesusg) [reply deploy/kiali/kiali_cr.yaml:629]: Yes you're right, let me do it

**Comment by jshaughn** [comment]
@aljesusg , why was this merged with no QE?

**Comment by jshaughn** [comment]
@aljesusg Your logic makes sense but I think QE should make the decision as to whether the community testing is sufficient.  They may want to perform some sort of regression tests, even if there is no need to test the feature.  I'll assign @mattmahoneyrh to take a look and make the decision.

**Comment by aljesusg** [comment]
You're right. My apologies for my fault. It's not going to happen again. Thanks

**Pull request merged**